api/envoy/config/filter/network/tcp_proxy/v2/tcp_proxy.proto

```diff
@@ -148,4 +148,8 @@ message TcpProxy {
   // The maximum number of unsuccessful connection attempts that will be made before
   // giving up. If the parameter is not specified, 1 connection attempt will be made.
   google.protobuf.UInt32Value max_connect_attempts = 7 [(validate.rules).uint32 = {gte: 1}];
+
+  // Optional configuration for TCP proxy hash. Source IP will be used for selecting hosts when the
```

Comment:
This seems a bit vague to me: how is source IP used to select a host? Does this apply to all load balancers? 

I would probably phrase it more as "the source IP will be used to compute the hash used by hash-based load balancing algorithms" or something to that effect.

Reply:
Done.